pom.xml

```diff
@@ -64,7 +66,8 @@
         <configuration>
           <scalaCompatVersion>${sbv}</scalaCompatVersion>
           <args>
-            <arg>-target:jvm-1.7</arg>
+            <arg>-target:jvm-11</arg>
+            <arg>-release:11</arg>
```

Comment:
these have been deprecated and replaced by `-java-output-version` in scala 3

Reply:
I think only target was deprecated some time ago, but `-release` seems to be the current param name: https://docs.scala-lang.org/scala3/guides/migration/options-lookup.html#standard-settings